isamples_metadata/SESARTransformer.py

```diff
@@ -71,6 +71,15 @@ class MaterialCategoryMetaMapper(AbstractCategoryMetaMapper):
     _equalsBiogenicMapper = StringEqualityCategoryMapper(
         ["Macrobiology>Coral>Biology", "Coral>Biology"], "Biogenic non-organic material"
     )
+    _equalsNaturalSolidMapper = StringEqualityCategoryMapper(
+        ["Natural Solid Material"], "Natural Solid Material"
+    )
+    _equalsMixedMapper = StringEqualityCategoryMapper(
+        ["Mixed soil, sediment, rock"], "Mixed soil, sediment, rock"
```

Comment:
Just want to double check -- the value here is the single comma-delimited string "Mixed soil, sediment, rock" and not an array of three strings.

Reply:
Yes, this is a new label added.